frontend/app/user/people-list.component.js

```diff
@@ -32,6 +32,20 @@ export default class PeopleList extends React.Component {
     });
   };
 
+  deleteCase = (person, kase, e) => {
+    e.preventDefault();
+    const array = [...this.state.cases];
+    remove(array, item => {
+      return item === kase;
+    });
+    this.setState({ cases: array });
+    context.setContext({
+      activePerson: "",
+      activeCase: ""
```

Comment:
Do we always need to reset the active case when a case is deleted? Can't we keep the activeCase if it wasn't the one that was deleted?

Reply:
We want to reset `activePerson` and `activeCase` only when active case is deleted. I'll add in a statement that checks for that.

Reply:
👍